### PR TITLE
feat(SPV-1277): add admin status and stats to new API

### PR DESCRIPTION
**Pull request by dorzepowski**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: actions/admin/routes.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ func RegisterRoutes(handlersManager *handlers.Manager) {
 	adminGroupOld := handlersManager.Group(handlers.GroupOldAPI, "/admin")
 
 	adminGroupOld.GET("/stats", handlers.AsAdmin(stats))
-	adminGroupOld.GET("/status", handlers.AsAdmin(status))
+	adminGroupOld.GET("/status", handlers.AsAdmin(statsOld))
```
Comment (dzolt-4chain) [inline actions/admin/routes.go:12]: It should be statusOld not statsOld right?
1. Please change statsOld to statusOld
2. or please change the path from /status to /stats

**Review by dzolt-4chain — CHANGES_REQUESTED** [review]

File: actions/admin/stats.go [hunk 2]
```diff
@@ -9,15 +9,36 @@ import (
...
+
+// stats will return the statistics of the spv-wallet
+// Get stats godoc
+// @Summary		Get stats
+// @Description	Get statistics of the spv-wallet
+// @Tags		Admin
+// @Produce		json
+// @Success		200	{object} response.AdminStats "Stats for the admin"
+// @Failure 	500	"Internal Server Error - Error while fetching admin stats"
+// @Router		/api/v1/admin/stats [get]
```
Comment (chris-4chain) [inline actions/admin/stats.go:41]: Minor. 

It seems like you could align when the second "column" starts.

File: actions/admin/stats.go [hunk 3]
```diff
@@ -9,15 +9,36 @@ import (
...
+		spverrors.ErrorResponse(c, err, reqctx.Logger(c))
+		return
+	}
+
+	contract := mappings.MapToOldAdminStatsContract(stats)
+	c.JSON(http.StatusOK, contract)
+}
+
+// stats will return the statistics of the spv-wallet
+// Get stats godoc
```
Comment (chris-4chain) [inline actions/admin/stats.go:34]: The "godoc" keyword is not needed, I guess.

**Review by chris-4chain — APPROVED** [review]

File: actions/admin/routes.go (hunk 1 above)
Comment (dorzepowski) [reply actions/admin/routes.go]: thanks for catching this 👍

**Review by mgosek-4chain — APPROVED** [review]

**Review by dzolt — APPROVED** [review]

**Review by dzolt-4chain — APPROVED** [review]

**Pull request merged**